src/Illuminate/Database/Schema/Blueprint.php

```diff
@@ -450,6 +451,10 @@ public function char($column, $length = 255)
      */
     public function string($column, $length = 255)
     {
+        if ($this->charset == 'utf8mb4' && $length == 255) {
```

Comment:
I'm so sorry. I forgot add the charset parameter of the construct function in the Blueprint class.